src/Tests/Features/OfferMatching/ListTests.cs

```diff
@@ -91,6 +91,33 @@ public async Task GivenAnyAuthenticatedUser_WhenPlayerIsMatched_ThenPlayerIsMove
         result.YearAcquired.Should().Be(DateTime.Today.Year);
         result.ContractValue.Should().Be(int.MaxValue);
     }
+
+    [Fact]
+    public async Task GivenAnyAuthenticatedUser_WhenPlayerHasNoBids_ThenPlayerIsMovedToUnsignedStatus()
```

Comment:
I’d drop all the asserts except the minimum bid and change this to “then has a contract value of one” or something?